test/test_local.py

```diff
@@ -34,6 +34,21 @@ def test_planes(request, run_in_tmpdir):
     diff_gold_file('plane.jou')
 
 
+@reset_openmc_ids
+def test_nested_spheres(request, run_in_tmpdir):
+    inner_sphere = openmc.Sphere(r=10.0)
+    middle_sphere = openmc.Sphere(r=20.0)
+    outer_sphere = openmc.Sphere(r=30.0)
+
+    inner_cell = openmc.Cell(region=-inner_sphere)
+    middle_cell = openmc.Cell(region=-middle_sphere)
```

Comment:
```suggestion
    middle_cell = openmc.Cell(region=+inner_sphere & -middle_sphere)
```
Shouldn't these be nested?